foreman/miner-watch

```diff
@@ -30,93 +30,16 @@ check_miner() {
+    if [[ -f "$PM_FILE" ]]; then
         # Extract information from the digest
-        MINER=$(jq -r .foreman.name $DIGEST_FILE)
-        API_PORT=$(echo $(jq -r .foreman.api_ext $DIGEST_FILE) | grep -Eo "[0-9]{4,5}")
+        MINER=$(jq -r .foreman.name $PM_FILE)
+        API_PORT=$(echo $(jq -r .foreman.api_ext $PM_FILE) | grep -Eo "[0-9]{4,5}")
     else
         # Attempt to parse the command line for the information.  This should all be removed
         # once everything is migrated to the new pluggable miner design
         case $MINER in
-            "anxccminer")
-                MINER="ccminer-phi"
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "asccminer")
-                MINER="ccminer-alexis78"
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "bminer")
-                API_PORT=$(pgrep -afn "$MINERS" | grep -Eo "api 127.0.0.1:[0-9]{4,5}" | cut -d '=' -f2 | cut -d ':' -f2 | xargs)
-                ;;
-            "claymore")
-                MINER="claymore-eth"
-                API_PORT=$(pgrep -afn "$MINERS" | grep -Eo "mport [0-9]{4,5}" | cut -d ' ' -f2 | xargs)
-                ;;
-            "cryptodredge")
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "dstm")
-                API_PORT=$(pgrep -afn "$MINERS" | grep -Eo "telemetry=127.0.0.1:[0-9]{4,5}" | cut -d '=' -f2 | cut -d ':' -f2 | xargs)
-                ;;
-            "ewbf")
-                API_PORT=$(pgrep -afn "$MINERS" | grep -Eo "api 127.0.0.1:[0-9]{4,5}" | cut -d '=' -f2 | cut -d ':' -f2 | xargs)
-                ;;
-            "ktccminer")
-                MINER="ccminer-klaust"
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "kxccminer")
-                MINER="ccminer-xevan"
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "lolminer")
-                API_PORT=$(pgrep -afn "$MINERS" | grep -Eo "apiport=[0-9]{4,5}" | cut -d '=' -f2 | xargs)
-                ;;
-            "msftccminer")
-                MINER="ccminer-msft"
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "naccminer")
-                MINER="ccminer-nanashi"
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "phoenixminer")
-                MINER="phoenix"
-                API_PORT=$(pgrep -afn "$MINERS" | grep -Eo "cdmport [0-9]{4,5}" | cut -d ' ' -f2 | xargs)
-                ;;
-            "silentminer")
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "sp_suprminer")
-                MINER="suprminer-sp-hash"
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "spccminer")
-                MINER="ccminer-sp-hash"
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "suprminer")
-                API_PORT=$(get_ccminer_port)
-                ;;
-            "tpccminer")
```

Comment:
I don't think this miner has a pm file yet, so this is still needed.

Reply:
Added TPccminer pm, please check,  https://github.com/papampi/nvOC_miners/pull/30/commits/6ee6172b7eeeb2c3c0c56cac6b6949636c57ecfe